crates/ruff_wasm/src/lib.rs

```diff
@@ -262,7 +261,7 @@ impl Workspace {
     }
 
     pub fn tokens(&self, contents: &str) -> Result<String, Error> {
-        let program = ruff_python_parser::parse_module(contents)?;
+        let program = parse_unchecked(contents, Mode::Module);
 
```

Comment:
This is just maintaining the existing behavior where the playground displays the token stream even if it contains syntax errors.